### DISCUSSION TITLE
[转贴]评论 | 程晓农：中国房贷断供潮的症结何在？

### DISCUSSION THREAD

**Discussion by TALK1949** [WeNeedHome/SummaryOfLoanSuspension#946] [General]
原文出处：[评论 | 程晓农：中国房贷断供潮的症结何在？（自由亚洲）](https://www.rfa.org/mandarin/pinglun/chengxiaonong/cxn-07192022102047.html)

中国的房地产市场让买房者的利益受到损害，终于导致全国性的房贷断供潮。究其原因，这是中国的制度之害。在立法、监管的各个层面，政府都站在房地产公司一边，为了维护房地产泡沫，不惜伤害消费者的利益，这就是发生房贷断供潮的症结所在。

一、新房预售制到了中国就出“篓子”

购买新房，经常会是预售的。新房预售制本身并没有问题，美国和台湾都有新房预售制。

在美国，订购新房子，房地产公司通常会让买主付出数千到一万美元的押金（相当于预付款），然后与买主签约；当新房的地基通过当地市政府建筑管理部门的检验后，初步构架便搭建起来，这时房地产公司会要求买主再支付一笔数万美元的工程款；到了房子完成全部内装修，所有家用电器都按照购房合约安装完成并经过检测，买主才进入自己的房子实地观察、仔细查验，由房地产公司派专人详细解释各项设施的使用和注意事项；最后是双方律师会面，买方支付剩余未付的房款（或者是买主开户银行的现金支票，或者是抵押贷款银行开列的支票），房地产公司则交出新房钥匙，购房合约执行完毕。剩下的是，在一年保修期内，房地产公司仍有义务免费为买主修理室内出现的各种瑕疵。这样的新房预售制运行多年，极少发生房地产公司和买主之间的纠纷。

台湾也实行新房预售制，做法与美国大体相似。新房的买主交了首付之后，剩余的大部分房款也是新房交付时才支付。由于台湾是民主制度，所以在民意压力下，政府更关注如何保护新房买主的利益。在台湾，依照相关法规，如果建商资金周转不灵，拖延施工，或者老板落跑，买主可以按照合约请求解约，向建商要求支付惩罚性违约金损失；同时，建商还要退还买主已经交付的款项。而且，中华民国內政部也有規定，建案如果延期交屋，每一天要补偿房子总价的万分之五给买主。也就是说，如果房子交付延迟两百天，建商给买主的补偿相当于房价的百分之十，这就把建商的利润基本赔光了。

中国1983年最早在深圳引入新房预售制。1994年，中国的《城市房地产管理法》正式确立商品房预售制度，此法律由1994年8届全国人大常委会第8次会议通过，1995年1月1日起施行；此后经人大常委会数次修改。目前适用的法律条文中，只有第4章“房地产交易”的第45条“商品房预售”中有这样一句空话，“商品房预售所得款项必须用于有关的工程建设”；但在第6章“法律责任”中，立法时却“忘了”规定，房地产公司如果违反第45条那句话时，是否要承担任何法律责任。也就是说，房地产公司若把商品房预售所得款挪出他用，在中国似属不当，却不需要付出违法代价，因为法无惩处条文。这就是这次中国因大量烂尾楼而造成房贷“断供潮”的原因之一。

中国的立法特点是法条用语政策化，当局这样做的目的是，为政策的随时调整留下足够的空间。而中国的政策执行象“橡皮筋”，于是房地产公司的“商品房预售所得款项”，本来依法“必须用于有关的工程建设”，但房地产公司不这样做，法律上却不予惩处，这个空洞的“必须”结果就变成了事实上的“不必”。从这里可以看出，在住房预售制方面，台湾和中国的差别在于，中国的法规有利于房地产公司，而台湾的法规有利于住房的买主。这本质上是民主政体与官商勾结的专制政体的差别。

二、中国特色的预售房“超前交款”

美国和台湾的新房预售制只要求住房买主在新房交付之前支付有限的首付，实行这种购房资金的支付模式，一般不会发生房地产公司留下“烂尾楼”的局面。因为如果房地产公司不能按时交房，损失自负。而中国却有一个自己特色的预售房“超前交款”的特殊做法，造成了政府保护房地产公司、而不管买房者可能吃亏的局面。

所谓的“超前交款”，是我从买主角度提出的概念；在中国的房地产业，这种做法被称为“超前回款”，意思是，房子还没盖好，房地产公司就预先把房款收足了。中国的开发商实际上是把预售房等同于已建房来销售，虽然房子根本还没完工，就要买主交足购房款。而房主交足购房款之后，实际上就失去了检查交工房产、或者与房地产公司谈判的能力。

在中国，新房预售制准许期房项目的主体结构封顶时，购房者就在首付的基础上申请按揭贷款；理论上，首付以及从银行申请的按揭贷款要存放在指定监管账户，根据施工、竣工的进度分批由开发商提取。甚至大量房地产公司要求买房者在首付款支付后的七天内，向与房地产公司合作的银行申请抵押贷款。为什么要买主申请房贷时，去房地产公司指定的银行去办理呢？因为那家银行和这个房地产公司是有勾结的，不但银行可能从房地产公司那里拿到好处，而且银行会悄悄地把买房者申请的房贷款项转给房地产公司。

这种房款“超前交款”的支付方式，可能让房地产公司占尽购房者的便宜。轻则交易不公平，所建房屋的许多细节达不到购房合约的要求，而购房者只能逆来顺受；重则房地产公司把购房者的房贷款项挪用到其他项目，导致购房者已足额付款的建房工程成为烂尾楼。

有人因此提出，这些问题都是商品房预售制造成的，希望全面取消预售制。其实，并不是预售制造成了中国房地产业的烂尾楼问题，因为国外的预售制并没有这样的问题；真正的罪魁祸首是中国特色的预售房“超前交款”支付模式，只要把预售制的房款支付方式改成国际通行的交钥匙付全款模式，烂尾楼问题就解决了。2019年下半年，《华夏时报》记者曾就此事采访了一位接近政策部门的专家，此人表示，那样做会对房企的冲击太大，多数企业无法承受，因此当时还不是时候。这表明，中国政府为房地产公司设想得面面俱到，唯独“忘了”买房者的利益需要保护。这是疏忽，还是故意，其实一目了然。

三、“监管”下预售房资金“开溜”

虽然买房者申请到的房贷资金是暂时存在贷款银行的专门账户里，但是，很多贷款银行本来就是由房地产公司指定的，这些银行与房地产公司有合作关系。与此同时，名义上，当地的房地产管理部门有监督责任，不能让房地产公司根据自己的资金需要把房贷资金拿去随便用。但是，中国的中央政府和立法机构放弃了实行统一法令、加强监管这个选项，而是让各地的房地产管理部门自行其是。于是，在似有似无、名有实无的各地房地产管理部门的“监管”之下，房主购买预售房的房贷资金就“开溜”了。

其中的奥秘在于，无论是相关银行，还是各地政府的房地产管理部门，都与房地产公司有暗中勾结。它们优先考虑的是房地产公司的需要，而不是贷款人的资金安全。只要房地产公司把其中的关节打通（行贿），银行里房地产贷款资金的被“监管”账户就变成了房地产公司的“资金池”。最常见的方式是，开发商从供应商那里取得似乎与建房工程相关的采购发票，凭此到银行就可以提取监管之下的房贷资金，但实际上房地产公司并没真正把这些资金用于施工，更没把住房项目竣工，这就是“烂尾楼”的由来。而更糟糕的情况是，银行在当地房地产管理部门的默许下，根本不把预售房的房贷资金放在监管账户里，而是直接转到了开发商的账上。

那房地产公司急于拿到房贷资金，要用到什么地方去？实际上，许多房地产公司是用甲住房项目买主们的房贷资金，去购买乙项目的开发用地，而这笔钱最后就进入了地方财政的口袋。这样就很好理解了，为什么各地政府的房地产管理部门故意不履行对房贷资金的监管责任，履行那种责任其实是自断本地政府的财路。中央政府不懂这一点吗？当然不是，北京是故意把“鱼”交给“猫”们来“监管”，其结果可想而知。

于是，各地房地产管理部门“监管”银行的房贷资金“专户”时，比较凭良心一点的，会按照工程建筑总投资预算的实施进度，上浮一定的百分点，让银行在“专户”里留存部分预售房被“监管”的资金，其余的钱转给房地产公司；有些地方则把预售资金的一个不大的比例留存在银行的“专户”里，其余部分转给急于用钱的房地产公司；比较恶劣的情形是，房地产管理部门根本不过问需要依法监管的预售房贷款专户资金，听凭银行把资金全数交给房地产公司。

而银行也是有几种做法。最常见的方式是，开发商随便提供一些开支发票，就可以从银行提取被“监管”的资金，至于房地产公司实际上是否施工、是否竣工，银行就不管了。更有甚者，银行根本不把预售房的房贷资金放进监管账户，而是直接转到开发商的账上。

如果政府和银行严格执行预售房的资金监管，本来完全可以保障项目完工，但大部分房地产公司实际上是把这些资金普遍用在土地市场上，去购买别的地块。所以，近年来各地政府拍卖土地给房地产公司时，所收到的买地收入当中，将近六成的钱是来自房地产公司从银行拿到的房主们购买先前销售的预售房贷款。也就是说，房贷资金成了地方政府土地收入的财源。

就这样，预售房买主们的宝贵钱财，被地方政府、银行和房地产公司给合伙骗走了。这些钱之所以回不来，是因为它已经成了地方政府财政收入的一部分，被当地政府花出去了；而房地产公司手头的预售房是没法完工的，这些公司只捏着几块没钱开工的新地块。

四、官方打击预售资金“跑冒滴漏”？

房地产公司想从房地产萧条和资金链困境当中跳出来的花招是，通过玩“击鼓传花”，让新地块上策划出来的纸上建房项目的买主们提供房贷资金，用来完工先前已售出的预售房。

新房的买主们听着房地产公司炒作新房的“鼓声”，加入这场“游戏”，却不知道什么时候“鼓声”停下来时，传到自己手里的“花”，其实是把自己的荷包“炸”得粉碎的“炸弹”。全国性房贷断供潮的房主们只知道自己买的房子没影儿了，却未必明白自己的钱“弯弯绕”到哪儿去了；他们若明白了这个“局”，恐怕就更着急了。

预售房资金有严重的“跑冒滴漏”，地方政府心里跟明镜似的，但它们“鼓着腮帮子看四方”，一脸无辜的样子。而中央政府呢，它也是揣着明白装糊涂，假模假式地“打击”了一番“挪用预售资金”，那是两年前的事儿了。

2020年，中央政府就意识到了“烂尾楼”的资金根源问题，因此要求各地政府要加强预售房资金的严格监管，要让每个住宅项目的资金“专款专用”，不少地方的政府也相应出台或修订了相关规定。虽然各地的法规细节不同，但都要求开发商必须协助购房人把房价款直接存进项目监管账户，不得直接收取房价款，并强调必须“专款专用”。

结果怎么样呢？最近的全国性房贷断供潮给出了回答。如果两年前住宅项目预售资金真的做到专款专用了，两年后，那时候的预售房早就盖完交付给房主了；如果“专款专用”了，也就没了预售资金的“跑冒滴漏”，那怎么会十几个省市到处都是“烂尾楼”呢？很显然，两年前政府的那次“打击挪用预售资金”之举，不过是一次“猫盖屎”行动，得益者未被追究，受害者继续受害。

最近中国大批烂尾楼盘业主采取了集体行动，宣布停止偿还房贷，涉及约20个省分、近70个市和区。这股风潮还在中国继续蔓延，每天都有新的住宅项目的住房买主们加入这项行动。目前，中国政府似乎还没找到解开这个死结的有效办法，但有一个结果是可以清晰地预料到的，那就是中国的房地产业面临着巨大的打击，消费者对各家房地产公司的不信任感快速上升。

接下来，不管当局如何应对，但消费者对买房的抵制已经自然形成了，谁也不想加入那个集体拒绝支付房贷的行列当中去；而结果就是，房地产销售和预售房制度崩盘了。房地产公司连现有楼盘都无力完工，更不可能挖掘资金来源去盖更多房子，用现房销售去吸引顾客。

那结果就会表现为，房地产业进入垮塌状态，而以房地产业为支柱的中国经济也陷入了困境。虽然谁也没想到，撕裂中国经济的房地产泡沫，居然是从买房者的如此集体行动开始的，但是“种祸得祸”这四个字如今是体验在北京当局身上了。中国政府现在更担心的是，房地产业的崩塌正在加深中国潜在的银行业金融危机。

（文章只代表特约评论员个人的立场和观点）

**Comment by Accelerator6666** [comment]
其实谁都知道问题在哪里啊，但是他们都集体沉默了，为什么沉默？还不是为了他们能喝到的那口汤

**Reply by xiazhifengtest** [reply]
这事银行和地产上早勾结好了，大家都知道这样违规，但是大家都知道这么干挣钱

**Reply by TALK1949** [reply]
银行和政府相关监管都对地产商睁一只眼闭一只眼，其实除了买房子的人，政府、银行、地产商都有责任，不过神奇就在这里，最后承受损失的却是屁民。

**Reply by wedxz** [reply]
愚昧的百姓,花最多的钱当最大的冤大头,还乐在其中

**Reply by dongxiangcheng** [reply]
不是不想，而是不敢，怪谁也不敢怪ZF，成本太高！就怪怪开发商，怪怪银行吧

**Comment by EricWong1202** [comment]
谁来帮我反驳一下这篇文章啊?我看了这篇文章之后不知道怎么反驳。

这篇文章的主要观点是:中国房贷断供潮是中国的制度之害。在立法、监管的各个层面，政府都站在房地产公司一边，为了维护房地产泡沫，不惜伤害消费者的利益，这就是发生房贷断供潮的症结所在。

**Reply by wedxz** [reply]
说的没毛病啊

**Reply by daiyingwo** [reply]
虽然整体上有一些主观色彩，但是基本骨架没啥毛病

**Reply by tingbiaolab** [reply]
看起来还是有道理的，多大的因必有多大的果。

**Comment by h9j6k** [comment]
这文章确定不是隔靴挠痒？？？？？？

维基百科１９９４年分税制条目说的很清楚了，https://zh.wikipedia.org/wiki/1994%E5%B9%B4%E5%88%86%E7%A8%8E%E5%88%B6%E6%94%B9%E9%9D%A9

摘网友评论，为什么14亿人奋斗出了高房价+生育率崩溃+灵活就业+经济危机? - 回复 by @weibao
https://pincong.rocks/question/item_id-526292

----
维基百科上面关于分税制的条目，指出房价暴涨的根源就是中央和地方争抢国库税收份额，分赃不均，大打出手，所以地方用土地财政来对付中央。连带造成的副作用就是神仙打架，老百姓遭殃，普通人没有参与制定任何政策，但是要为中央和地方政府打架造成的高企房价卖身３０年。还是那句话，上面当官的，花别人的钱，给自己的办事，绝对对别人心狠手辣，恨不得可以多压榨一点。

ｓｔｅｉｎ　ｒｉｎｇｅｎ的书上说中国的实际税率在５０％，也就是说中国的老百姓上交北欧国家水平的税收，返还的福利大概和没有福利的国家差不多水平，站在个人的角度，傻子都知道是强迫的亏本买卖。但是这种不合理的税收当官的怎么向老百姓解释啊，开动媒体，央视新华社各种中国梦，各种奋斗发家史，洗洗洗洗，洗着洗着，大家就以为自己是在为自己的将来奋斗，自己是为自己的幸福拼搏。所以，他们最拿手的就是宣传，让大家以为他们是在做他们宣称的为人民服务的事业，演得真像，大家也就信了，再说，就算不信，也不能把他们怎样，他们有枪。

根本没有什么ｇｏｖｅｒｎｉｎｇ，根本没有什么政治智慧，只是一群在挂着国家部门牌子大楼里上班的骗子。

**Reply by Streaking-snail** [reply]
才50%？算上40%的五险了吗，漂亮国社保也算进税里的哦

**Reply by zxcs001** [reply]
在国外网站打字，还打一堆空格和英文自我和谐。真是懦弱胆小到骨子里面了……  

直接把事情说明白，有那么难么？

就是共产党红色血脉把老百姓当韭菜往四里面割，让全民背上几十年负债，然后把资产全部放到瑞士银行呗。

直接说这些东西很难么？

一大堆中国人不把自己当人罢了，都是些牛马，牲畜，作为人唯一的价值就是提供生产力，为了红色贵族积累在国际上能够交换的财富罢了。你拿的那些红票票都是些代金券而已，为什么正儿八经的红色血脉都不屑这些东西？因为他们知道，要真是印钞票的话，他们想印多少就印多少，这玩意可以分分钟变的一分钱都不值。所以正儿八经的常委之类的，都是直接持有资产和美金。这些玩意才是实打实有价值的东西。

**Reply by itsoursduty** [reply]
> 在国外网站打字，还打一堆空格和英文自我和谐。真是懦弱胆小到骨子里面了……
> 
> 直接把事情说明白，有那么难么？
> 
> 就是共产党红色血脉把老百姓当韭菜往四里面割，让全民背上几十年负债，然后把资产全部放到瑞士银行呗。
> 
> 直接说这些东西很难么？
> 
> 一大堆中国人不把自己当人罢了，都是些牛马，牲畜，作为人唯一的价值就是提供生产力，为了红色贵族积累在国际上能够交换的财富罢了。你拿的那些红票票都是些代金券而已，为什么正儿八经的红色血脉都不屑这些东西？因为他们知道，要真是印钞票的话，他们想印多少就印多少，这玩意可以分分钟变的一分钱都不值。所以正儿八经的常委之类的，都是直接持有资产和美金。这些玩意才是实打实有价值的东西。

个人觉得在github打字也不算安全，还是小心点为好

**Comment by lance111l** [comment]
房地产是大支柱  国家会保证的  但是小企业盲目上马  资金断裂 导致烂尾楼  例外
